README.md

```diff
@@ -220,6 +219,16 @@ desc 'Get all kittens!', {
 get '/kittens' do
 ```
 
+## Specify endpoint details
+To specify further details for an endpoint, use the `detail` option within a block passed to `desc`:
+
+``` ruby
+desc 'Get all kittens!' do
+  detail 'this will expose all the kittens'
+end
```

Comment:
well `notes` is grape-swagger specific, `detail` is supported by grape  in general. so `notes` should be migrated to `detail`. What do you think to write?